capability/LICENSE

```diff
@@ -0,0 +1,25 @@
+Copyright 2023 The Capability Authors.
```

Comment:
And Moby authors

Reply:
We briefly discussed options here; we could even choose to make it just "The Authors", which is the most generic form; ultimately, copyright of any code change belongs to the author (what project that code change was made in, doesn't really matter there).

I guess the only benefit of mentioning the project is "where did it originate from" in cases where the code is forked by others.

Reply:
I think "the Capability Authors" covers everyone who contributes to this repository.